python/lsst/ip/isr/isrTask.py

```diff
@@ -1225,3 +1225,50 @@ def getSaturation(self):
 
     def getSuspectLevel(self):
         return float("NaN")
+
+
+class IsrWrapperTaskConfig(pexConfig.Config):
+    isr = pexConfig.ConfigurableField(target=IsrTask, doc="Instrument signature removal")
+
+## @addtogroup LSST_task_documentation
+## @{
+## @page IsrWrapperTask
+## @ref IsrWrapperTask_ "IsrWrapperTask"
+## @copybrief IsrWrapperTask
+## @}
+
+
+class IsrWrapperTask(pipeBase.CmdLineTask):
+    """!
+    @anchor IsrWrapperTask_
+
+    @brief Task to wrap the main IsrTask to allow camera specific retargeting.
+    """
```

Comment:
Please make the new docstrings numpydoc, otherwise we'll just have to convert them later.

Also, can you please expand on this documentation a bit? I don't really understand what makes it different from the original `IsrTask`.

Reply:
Docstrings updated to numpydoc.  I didn't use a sufficiently new comparison to get the new format.

Docstring also amended to include information on what this change does, and why it's needed.